pkg/util/consts.go

```diff
@@ -38,3 +50,15 @@ const (
 	HCONSWebhookPath            = "/mutate-ns-hco-kubevirt-io"
 	WebhookPort                 = 4343
 )
+
+type AppComponent string
+
+const (
+	ComputeComponent    AppComponent = "compute"
+	StorageComponent                 = "storage"
+	ImportComponent                  = "import"
+	NetworkComponent                 = "network"
+	MonitorComponent                 = "monitor"
```

Comment:
I think "monitoring" or maybe "metrics" could be a better name than "monitor".

Reply:
After the discussions in chat, I updated this part as "monitoring". Also changed the names of enums like the ServiceType naming of k8s https://github.com/kubernetes/kubernetes/blob/ddf3eb5a1877338da806c2be15b53f456b6342c9/pkg/apis/core/types.go#L3446